vagrant/config/cert

```diff
@@ -0,0 +1,63 @@
+[init] using Kubernetes version: v1.12.3
```

Comment:
plase do not commit changes in the vagrant folder

Reply:
it is Ok I have added the vagrant folder to the .gitignore